### DISCUSSION TITLE
For anonymous OJS users: ui.js (which generates locale strings) is not available if allowed_hosts is not set

### DISCUSSION THREAD

**Discussion by withanage** [pkp/pkp-lib#10491] [Proposals]
#### Issue

- Since OJS 3.4, the setting allowed_hosts is a mandatory field in the file config.inc.php.
- After OJS 3.4 , the following setting allowed_hosts = ‘[\’localhost\‘]’ is already included in the `config.TEMPLATE.inc.php `file , therefore it would not create an unauthorized error for not logged users.

If a server administrator moves an older config.file from an older installation prior to OJS 3.4, `allowed_hosts = ‘’` or the correct host setting must be added to the config.inc.php file. Recommended is the exact host name as described in the `config.TEMPLATE.inc.php

**Comment by withanage** [comment]
FYI @jardakotesovec

**Comment by asmecher** [comment]
@withanage, this is already included in the OJS 3.4 release notes; see: https://github.com/pkp/ojs/blob/main/docs/release-notes/README-3.4.0#L24
